machine/v1beta1/types_gcpprovider.go

```diff
@@ -143,10 +147,14 @@ type GCPMachineProviderSpec struct {
 	// +optional
 	ShieldedInstanceConfig GCPShieldedInstanceConfig `json:"shieldedInstanceConfig,omitempty"`
 
-	// confidentialCompute Defines whether the instance should have confidential compute enabled.
-	// If enabled OnHostMaintenance is required to be set to "Terminate".
+	// ConfidentialCompute Defines whether the instance should have confidential compute enabled or not, and the confidential computing technology of choice.
```

Comment:
The previous godoc start was correct in the field name formatting. In OpenShift we try to use the serialized name (aka the name in the JSON tag) since that is what users will see when using something like `oc explain ...` to explore and learn about the API.

Reply:
Just now realizing that `oc explain ...` wouldn't be used here because this is a machine type that I don't think produces a CRD. Still worth sticking to the OpenShift conventions here.

Reply:
Yes, my bad. Updated.